frontend/src/metabase/query_builder/components/expressions/ExpressionEditorSuggestions.jsx

```diff
@@ -97,25 +99,32 @@ export default class ExpressionEditorSuggestions extends React.Component {
             const { icon } = suggestion;
             const { normal, highlighted } = colorForIcon(icon);
 
-            return (
-              <React.Fragment key={`suggestion-${i}`}>
-                <ListItemStyled
-                  onMouseDownCapture={e => this.onSuggestionMouseDown(e, i)}
+            const key = `$suggstion-${i}`;
+            const listItem = (
+              <ListItemStyled
+                onMouseDownCapture={e => this.onSuggestionMouseDown(e, i)}
+                isHighlighted={isHighlighted}
+                className="flex align-center"
+              >
+                <Icon
```

Comment:
Nit, this generic Icon can also be made into a styled component, getting the just props needed to style it.

"Markup" would be slightly leaner, and presentation would be in the styled file.

Reply:
Will punt on this; have a lot of these popover implementations to get through -- hope you don't mind :)